core/core-impl/src/main/java/ai/timefold/solver/core/impl/domain/solution/cloner/gizmo/GizmoSolutionClonerFactory.java

```diff
@@ -28,7 +28,8 @@ public static <T> SolutionCloner<T> build(SolutionDescriptor<T> solutionDescript
                     ") the classpath or modulepath must contain io.quarkus.gizmo:gizmo.\n" +
                     "Maybe add a dependency to io.quarkus.gizmo:gizmo.");
         }
-        return GizmoSolutionClonerImplementor.createClonerFor(solutionDescriptor, gizmoClassLoader);
+        return new GizmoSolutionClonerImplementor().createClonerFor(solutionDescriptor,
```

Comment:
Why not:

```suggestion
        return GizmoSolutionClonerImplementor.createClonerFor(solutionDescriptor,
```

Seems like the instance creation is just boilerplate, it can happen internally. People need not know what's happening under the covers.

Reply:
See comment below